zap/src/main/java/org/parosproxy/paros/view/MainFrame.java

```diff
@@ -377,6 +379,7 @@ private JButton getShowAllTabsButton() {
                         @Override
                         public void actionPerformed(ActionEvent e) {
                             View.getSingleton().showAllTabs();
+                            Stats.incCounter("stats.ui.toolbar.button.tabs.showall");
```

Comment:
I'd suggest prefix with "maintoolbar", we have other toolbars that could also have stats in the future.